src/devices/video/gb_lcd.h

```diff
@@ -262,6 +256,8 @@ class sgb_ppu_device : public dmg_ppu_device
 	virtual void update_sprites() override;
 	virtual void update_scanline(uint32_t cycles_to_go) override;
 	void refresh_border();
+	
+	void sgb_vram_memcpy(uint8_t* dst, uint8_t start, size_t num_tiles);
```

Comment:
It's not a unary `*`, because it's not an operator... I'd dispute if this is closer to the parser, but I can see the consistency.